vhdbuilder/packer/components.json

```diff
@@ -11,6 +11,9 @@
       "downloadURL": "mcr.microsoft.com/oss/kubernetes/dashboard:*",
       "amd64OnlyVersions": [
         "v2.0.0",
+        "v2.0.0",
```

Comment:
why dup entries.

Reply:
there seem to be some issue in dedup process when there is a leading `v` in image tag
fixed this PR with a new commit for now